### ISSUE TITLE
Support for publishing multi-line text in `description`.

### ISSUE THREAD

**Issue by IceCodeNew** [github-release/github-release#112]
Command:
<pre>
# github-release release \
    --user cbdb-project \
    --repo haproxy_static \
    --tag "v${haproxy_latest_tag_name}" \
    --name "v${haproxy_latest_tag_name}" \
    --description "The released deb packages had been tested on Ubuntu 16.04 & Debian 8.\nIt is recommended to initiate a Systemd drop-in file for the haproxy that supports the sdnotify features of systemd, just issue command `mkdir -p '/etc/systemd/system/haproxy.service.d' && echo -e '[Service]\nRestartSec=6' > '/etc/systemd/system/haproxy.service.d/haproxy.conf'`\nFor the haproxy that **DOES NOT** support the sdnotify features of systemd, issue command `mkdir -p '/etc/systemd/system/haproxy.service.d' && echo -e '[Service]\nExecStart=\nExecStart=/usr/local/sbin/haproxy -W -f $CONFIG -p $PIDFILE $EXTRAOPTS\nType=\nType=forking\nRestartSec=6' > '/etc/systemd/system/haproxy.service.d/haproxy.conf'`"
</pre>

Turns out:
![image](https://user-images.githubusercontent.com/32576256/107588418-9b0c9f80-6c3e-11eb-8c23-d666c32dce4b.png)

Expected result:
<pre>
The released deb packages had been tested on Ubuntu 16.04 & Debian 8.
It is recommended to initiate a Systemd drop-in file for the haproxy that supports the sdnotify features of systemd, just issue command `mkdir -p '/etc/systemd/system/haproxy.service.d' && echo -e '[Service]\nRestartSec=6' > '/etc/systemd/system/haproxy.service.d/haproxy.conf'`
For the haproxy that **DOES NOT** support the sdnotify features of systemd, issue command `mkdir -p '/etc/systemd/system/haproxy.service.d' && echo -e '[Service]\nExecStart=\nExecStart=/usr/local/sbin/haproxy -W -f $CONFIG -p $PIDFILE $EXTRAOPTS\nType=\nType=forking\nRestartSec=6' > '/etc/systemd/system/haproxy.service.d/haproxy.conf'`
</pre>

**Comment by IceCodeNew** [comment]
`github-release` also removed the commands which are wrap by backticks, which also baffles me.

**Comment by kevinburke** [comment]
> github-release also removed the commands which are wrap by backticks, which also baffles me.

These are both issues with your shell - backticks are being evaluated as commands before github-release even sees the string to evaluate. Run any command, like "echo" or "cat" and you'll get the same result.

**Comment by IceCodeNew** [comment]
> > github-release also removed the commands which are wrap by backticks, which also baffles me.
> 
> These are both issues with your shell - backticks are being evaluated as commands before github-release even sees the string to evaluate. Run any command, like "echo" or "cat" and you'll get the same result.

Got it, thanks!

**Comment by IceCodeNew** [comment]
I was able to solve this problem by issuing:
```
echo -e 'The released deb packages had been tested on Ubuntu 16.04 & Debian 8.\nIt is recommended to initiate a Systemd drop-in file for the haproxy that supports the sdnotify features of systemd, just issue command `mkdir -p '/etc/systemd/system/haproxy.service.d' && echo -e '[Service]\nRestartSec=6' > '/etc/systemd/system/haproxy.service.d/haproxy.conf'`\nFor the haproxy that **DOES NOT** support the sdnotify features of systemd, issue command `mkdir -p '/etc/systemd/system/haproxy.service.d' && echo -e '[Service]\nExecStart=\nExecStart=/usr/local/sbin/haproxy -W -f $CONFIG -p $PIDFILE $EXTRAOPTS\nType=\nType=forking\nRestartSec=6' > '/etc/systemd/system/haproxy.service.d/haproxy.conf'`' | github-release release \
    --user cbdb-project \
    --repo haproxy_static \
    --tag "v${haproxy_latest_tag_name}" \
    --name "v${haproxy_latest_tag_name}" \
    --description -
```
